BCG-Gamma/facet
# Auto load data for readme

**Description**
**Updated**
Following conversation on conclusions from scaled sklearn data, data source has been updated to URL of original dataset at https://www4.stat.ncsu.edu/~boos/var.select/diabetes.tab.txt 
- Related changes in conclusions for BMI have been reverted with real values of BMI
- Notebook has been updated to align with Readme

**Original PR**
Readme and example notebook use a CSV file that was a part of the repository. This dataset is available with sklearn and this change automates dataload from sklearn instead of CSV. Key changes and description of commits are:
- **Loading data from sklearn instead of csv in readme:**  This commit removes read csv step and replaces it with loading bunch object from sklearn.datasets. To enhance readbility, column names of the native dataset have been changed too
- **Fixing what looks like typos in conclusions for synergy:** Conclusion of TC having synergistic relationship with other variables looked incorrect and misaligned. This has been updated to align with the visualisation
- **Changing column name from TSH to TCH:**  This is a change made in the markdowns; sklearn documentation shows TCH would be a better description of this column than TSH
- **Remove part of conclusion in simulation for BMI:** sklearn data is scaled, and therefore conclusion based on raw value of BMI is harder to interpret for the user. Since, the actual value of the conclusion is less important that the demo of the functionality, the part of the conclusion referring to raw value of BMI has been removed.
- **Updating diabetes notebook to align with readme:** Changes in Readme are replicated in the notebook in sphinx folder
- **Deleting diabetes.csv:** Since, the example does not need the csv anymore, the file has been removed

**How to test**
- Open notebook (this can be in your local) and use the code in updated readme
- The conclusions in readme, should be aligned with the results you see in your notebook
